frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -0,0 +1,332 @@
+<template lang="pug">
+page-section.c-section(:title='L("Invite links")')
+  template(#cta='')
+    .select-wrapper.c-select-wrapper(:class='ephemeral.selectbox.focused ? "focused" : ""')
+      select.button.is-small.is-outlined.c-select(
```

Comment:
Could you create a component for that selector? (out of scope but good to have if you have time)

Reply:
I will make it a re-usable component once the issue we have in [this thread](https://github.com/okTurtles/group-income-simple/pull/723#discussion_r336767822) is figured out. Would you have any suggestion on this issue as well? :)